core/src/test/resources/filter-tests.adoc

```diff
@@ -72,7 +86,24 @@ UNWIND [{
 AS props
 
 CREATE (p:Person)-[:WORKS_AT]->(c)
-SET p = props
+SET p = props;
```

Comment:
why adding test data, if you don't use it in the tests?

Reply:
In below tests, there are graphql queries which are filtering on node properties. So this "props" is required here.

But in my test case for "wrong variable name in where condition", props is not required. I will remove it from there.

Reply:
Removed unwanted data from my test case

Reply:
What I meant is that if you add test data, which is perfectly OK, then the test should also check if a particular query returns that data. This is tested using the following block:

```adoc
.GraphQL-Response
[source,json,response=true,ignore-order]
----
{
 "expected": "graphql response"
}
----
```

Reply:
Added Graphql Response